dts/bindings/display/panel/panel-timing.yaml

```diff
@@ -4,28 +4,24 @@
 # Common fields for panel timings
 # inherited from Linux panel bindings.
 
-description: |
-  Common timing settings for display panels. These timings can be added to
-  a panel under display-timings node. For example:
-
-  &lcdif {
-    display-timings {
-      compatible = "zephyr,panel-timing";
-      hsync-len = <8>;
-      hfront-porch = <32>;
-      hback-porch = <32>;
-      vsync-len = <2>;
-      vfront-porch = <16>;
-      vback-porch = <14>;
-      hsync-active = <0>;
-      vsync-active = <0>;
-      de-active = <1>;
-      pixelclk-active = <0>;
-      clock-frequency = <62346240>;
-    };
-  };
-
-compatible: "zephyr,panel-timing"
+#  Common timing settings for display panels. These timings can be added to
```

Comment:
Can child bindings not have a description? Not sure why this needs to be commented out.

Reply:
they can, but no documentation would get generated for it AFAIK, so I don't see the point, but it doesn't hurt anything, I guess, so I can put it back if you care enough. But do you know what would be the point?

Reply:
I guess there is no strong reason to do it either way. The only reason I can think of to use the `description` property would be if we did start generating the documentation in the future. It also follows the pattern we use for other child bindings, where we do add a description property